addons/website_blog/static/src/js/options.js

```diff
@@ -98,7 +98,10 @@ options.registry.BlogPostTagSelection = options.Class.extend({
      * @see this.selectClass for params
      */
     setTags(previewMode, widgetValue, params) {
-        this.tagIDs = JSON.parse(widgetValue).map(tag => tag.id);
+        if (!this._preventNextSetTagsCall) {
+            this.tagIDs = JSON.parse(widgetValue).map(tag => tag.id);
+        }
+        this._preventNextSetTagsCall = false;
```

Comment:
I am changing to 
```javascript
        if (this._preventNextSetTagsCall) {
            this._preventNextSetTagsCall = false;
            return;
        }
        this.tagIDs = JSON.parse(widgetValue).map(tag => tag.id);
```
I always prefer that form when possible but in this case it also avoids touching the history of the line we are less likely to refactor later.